src/Components/AreaCards.js

```diff
@@ -1,26 +1,23 @@
 import React from "react";
-import './AreaCards.css'
+import '../AreaCards.css'
 import { NavLink } from "react-router-dom";
-import bedroom from './bedroom.jpeg';
-import bathroom from './bathroom.jpeg';
-import kitchen from './kitchen.jpeg';
+import bedroom from '../bedroom.jpeg';
+import bathroom from '../bathroom.jpeg';
+import kitchen from '../Kitchen.jpeg';
```

Comment:
You jump between using double/single quotes and ending statements with/without semi-colons. For the sake of consistency, I recommend going back refactoring all the statements to use one or the other 
(i.e `' '` or `" "` and `;` or none.